DDCore/src/OpticalSurfaceManager.cpp

```diff
@@ -0,0 +1,157 @@
+//==========================================================================
+//  AIDA Detector description implementation 
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+// Author     : M.Frank
+//
+//==========================================================================
+
+// Framework include files
+#include "DD4hep/OpticalSurfaceManager.h"
+#include "DD4hep/detail/OpticalSurfaceManagerInterna.h"
+#include "DD4hep/ExtensionEntry.h"
+#include "DD4hep/Detector.h"
+#include "DD4hep/Printout.h"
+
+// C/C++ includes
+#include <sstream>
+#include <iomanip>
+
+using namespace std;
+using namespace dd4hep;
+
+/// static accessor calling DD4hepOpticalSurfaceManagerPlugin if necessary
+OpticalSurfaceManager OpticalSurfaceManager::getOpticalSurfaceManager(Detector& description)  {
+  return description.surfaceManager();
+}
+
+#if ROOT_VERSION_CODE > ROOT_VERSION(6,16,0)
```

Comment:
This would also be true for 6.16.01, would that be OK? or should it be just >= 6.17

Reply:
Point taken.